### PR TITLE
Clear auth token cookie on become_user and logout; update frontend tests

### LINKED ISSUE #227: Investigate why websocket messages are not reaching front-end during tests
When resolved, remove instances of `driver.refresh()` in tests that act as temporary workarounds.

### PR DESCRIPTION
**Pull request by acrellin**
Closes #227

### CODE REVIEW

File: skyportal/tests/frontend/test_tokens.py [hunk 1]
```diff
@@ -14,19 +14,20 @@ def test_add_token(driver, user, public_group):
...
+    token_id = view_only_token_created_by_fulluser
     driver.get(f'/become_user/{user.id}')
     driver.get('/profile')
     driver.wait_for_xpath(f'//input[@value="{view_only_token_created_by_fulluser}"]')
     driver.wait_for_xpath('//a[contains(text(),"Delete")]').click()
-    driver.refresh()
+    driver.wait_for_xpath(f'//div[contains(text(),"Token {token_id} deleted.")]')
     try:
-        driver.wait_for_xpath(f'//input[@value="{view_only_token_created_by_fulluser}"]')
+        driver.wait_for_xpath(f'//input[@value="{token_id}"]',
```
Comment (stefanv) [inline skyportal/tests/frontend/test_tokens.py]: @acrellin We'd prefer not to have a timeout-based test here.  Grab the table and ensure the token id is missing.

Comment (acrellin) [reply skyportal/tests/frontend/test_tokens.py]: @stefanv this is what we decided was the best approach when we were looking at this together, remember? Note that the timeout is set to 0, so it doesn't wait

Comment (acrellin) [reply skyportal/tests/frontend/test_tokens.py]: @stefanv it waits for the notification to display then immediately (without a wait period) makes sure this element doesn't exist

**Comment by stefanv** [comment]
Missed the `timeout=0`, all good!

**Pull request merged**